### PR TITLE
Connected Cluster Documentation Updates

### PR DESCRIPTION
**Pull request by ashnanze**
## Purpose

* ...

## Does this introduce a breaking change?

```
[ ] Yes
[ ] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Get the code

```
git clone [repo-address]
cd [repo-name]
git checkout [branch-name]
npm install
```

* Test the code

```
```

## What to Check
Verify that the following are valid
* ...

### CODE REVIEW

File: .github/workflows/README.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# CNAB Bundle Build 
+This Github workflow automates the building and publishing of the azure-vote CNAB (Cloud Native Application Bundle). 
+
+# Key Steps
+1. Checkout Repository
+- This step allows the workflow to access the code in the GitHub repository. The step uses actions/checkout@v3 to clone the code in the repository into the current workflow environment.
+
+2. Build CNAB Bundle
```
Comment (RamyasreeChakka) [inline .github/workflows/README.md]: Update text to "Build CNAB Bundle and Publish to ACR"

File: .github/workflows/README.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+- This step builds and publishes the CNAB. The step uses the action addnab/docker-run-action@v3 to run a Docker container with the specified parameters:
```
Comment (RamyasreeChakka) [inline .github/workflows/README.md]: This step builds CNAB and publishes it to Azure Container Registry (ACR).

File: .github/workflows/cnab-bundle-build.yaml [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+jobs:
+    build-and-publish:
+        runs-on: ubuntu-latest
+        environment: production
+        steps:
+            - name: Checkout repository
+              uses: actions/checkout@v3
+                
+            - name: Build CNAB Bundle
```
Comment (RamyasreeChakka) [inline .github/workflows/cnab-bundle-build.yaml]: Build CNAB Bundle and Publish to ACR

File: samples/k8s-connectedcluster-only/README.md [hunk 4]
```diff
@@ -34,9 +34,9 @@ Following is the guidance on the supportedClusterTypes:
 | Property      | Description |
 | -----------   | ----------- |
 | supportedClusterTypes       | Contains an object for each top-level cluster-type. Allowed types are – “managedClusters”, “connectedClusters”. "managedClusters" denotes Azure Kubernetes Service (AKS) clusters. "connectedClusters" denotes Azure Arc-enabled Kubernetes clusters. For each of these cluster types, specify distributions and unsupported Kubernetes versions for these distributions. If supportedClusterTypes is not provided, all distributions of ‘managedClusters’ will be supported by default. If supportedClusterTypes is provided, and a given top level cluster type is not provided, then all distributions and Kubernetes versions for that cluster type will be treated as unsupported. |
```
Comment (RamyasreeChakka) [inline samples/k8s-connectedcluster-only/README.md]: For Azure Kubernetes Service clusters, can you point to link https://learn.microsoft.com/en-us/azure/aks/what-is-aks
For Azure Arc-enabled Kubernetes clusters, can you point to link https://learn.microsoft.com/en-us/azure/azure-arc/kubernetes/overview

**Review by RamyasreeChakka — CHANGES_REQUESTED** [review]

File: .github/workflows/cnab-bundle-build.yaml (hunk 3 above)
Comment (ashnanze) [reply .github/workflows/cnab-bundle-build.yaml]: Updated!

**Review by RamyasreeChakka — APPROVED** [review]

**Review by RamyasreeChakka — APPROVED** [review]
LGTM

**Pull request merged**